### PR TITLE
Writing UI and eyes test file for HOC Emial Conversion Dialog

### PR DESCRIPTION
**Pull request by hannahbergam**
This PR adds a new feature file to capture the signed in and signed out views of the HOC email tutorial dialog. This is already partially covered by other ui tests (only for signed in users- there is no existing signed out coverage). 

## Warning!!

We have entered Pixel Lock for Hour of Code!

Computer Science Education Week will be happening from Dec 4 - Dec 10. Alongside this event, we will
be launching our new Hour of Code activity. Please consider any risk introduced in this PR that
could affect Dance Lab, instructions, saving and logging student progress, caching, or anything
related to the Hour of Code activities, old or new. Even small changes, such as a different button
color, are considered significant during this time. If this change will affect the new Hour of Code
activity in any way, join the morning change review to get your changes approved prior to merging.
Reach out to the Student Labs team for more details!

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1082

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Feature: HOC Guide Email Conversion Dialog
+
+  @eyes
+  Scenario: View and Close Dialog as signed in Instructor
+    When I open my eyes to test "signedIn"
```
Comment (bethanyaconnor) [inline dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature]: should this have a more descriptive name? I can't remember how this shows up on Applitools (may not be necessary to rename if the context is clear)

File: dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    Given I create a teacher named "Coach"
+    And I sign in as "Coach"
+    Then I am on "http://studio.code.org/s/alltheselfpacedplthings/lessons/1/levels/2"
```
Comment (bethanyaconnor) [inline dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature]: out of curiousity, why not just use allthethings?

File: dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  Scenario: View Dialog and send Email as signed out User
+    When I open my eyes to test "signedOut"
+    Given I am on "http://studio.code.org/s/allthethings/lessons/37/levels/2?noautoplay=true"
+    And I wait for the page to fully load
+    And I wait for 3 seconds
+    And I wait until I don't see selector "#p5_loading"
+    And I select age 21 in the age dialog
+    And I see no difference for "signedOutGuide"
+    And I press keys "Simone" for element "input#uitest-hoc-guide-name"
+    And I press keys "Simone@Biles.com" for element "input#uitest-hoc-guide-email"
```
Comment (bethanyaconnor) [inline dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature:30]: love this 😍

**Review by bethanyaconnor — APPROVED** [review]
A couple nits but otherwise looks good!

File: dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature (hunk 1 above)
Comment (hannahbergam) [reply dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature]: Good call- updated!

File: dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature (hunk 2 above)
Comment (hannahbergam) [reply dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature]: I was copying the levels used in the Age Dialog feature file lol but this is a good point- updated

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**